docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1560,6 +1560,18 @@ Default stratifications to all observers should include scenario and input draw.
     - Baseline; AI-assisted ultrasound scale-up; CPAP and ACS scale-up; CPAP, ACS, and AI-ultrasound scale-up 
     - Default
     - 
+  * - 20.1
+    - Hemoglobin refactor
+    - Bringing model up to date with the `updated hemoglobin module docs <https://github.com/ihmeuw/vivarium_research/pull/1830>`__ and fixing bugs in "Fix observer stratifications and multiple bugfixes" related to multiple instances of hemoglobin variables that were being inconsistently referenced by different simulation components (See outstanding model verification and validation issues table for full list)
+    - Baseline, MMS scaleup, and anemia screening scaleup scenarios
+    - Do not define a measure of "true first trimester hemoglobin exposure" in this version of the model. Otherwise, default.
+    -
+  * -
```

Comment:
Note @hussain-jafari I think this run should be numbered 21.0.1, but I haven't included that here because I think it's technically up to you now